guides/node-operators/run-a-super-world-computer-node.md

```diff
@@ -10,7 +10,7 @@ This guide will help you get a Super World Computer node up and running.
 1. Prepare [rollup.json](https://github.com/ethstorage/pm/blob/main/L2/assets/beta_testnet_rollup.json) and [genesis.json](https://github.com/ethstorage/pm/blob/main/L2/assets/beta_testnet_genesis.json).
 
 
-2. Follow the steps [here](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup) to build `op-node` and `op-geth`, the only difference is: use the `beta_testnet` branch of both [optimism](https://github.com/ethstorage/optimism/tree/devnet) and [op-geth](https://github.com/ethstorage/op-geth/tree/devnet) instead.
+2. Follow the steps [here](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup) to build `op-node` and `op-geth`, the only difference is: use the `beta_testnet` branch of both [optimism](https://github.com/ethstorage/optimism/tree/beta_testnet) and [op-geth](https://github.com/ethstorage/op-geth/tree/beta_testnet) instead.
```

Comment:
Is it related to the CGT change?

Reply:
No, a typo discovered by Qi [here](https://github.com/ethstorage/swc-doc/pull/9#discussion_r1926649197).